modules/nf-core/ribocode/ribocode/tests/main.nf.test.snap

```diff
@@ -0,0 +1,97 @@
+{
+    "test_ribocode_ribocode": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        [
+                            "test.txt:md5,89e5dc3e39fb52e98e921a379ee68b69",
+                            "test_collapsed.txt:md5,dbd709d42f94c6148bdbc0849dd028fb"
+                        ]
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test_collapsed.txt:md5,dbd709d42f94c6148bdbc0849dd028fb"
+                    ]
+                ],
+                "2": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        "test_ORFs_category.pdf:md5,a56356649534ea846fe7aa3aac6684ff"
```

Comment:
We probably shouldn't try to snapshot pdfs, they tend not to be stable over multiple machines because of differences in underlying libraries.

Reply:
Gone!